engines/advancedDetector.cpp

```diff
@@ -513,31 +503,56 @@ namespace Common {
 	DECLARE_SINGLETON(MD5CacheManager);
 }
 
-// Sync with engines/game.cpp
-static char flagsToMD5Prefix(uint32 flags) {
-	if (flags & ADGF_MACRESFORK) {
-		if (flags & ADGF_TAILMD5)
-			return 'e';
-		return 'm';
+
+static MD5Properties gameFileToMD5Props(const ADGameFileDescription *fileEntry, uint32 gameFlags) {
+	if(fileEntry && fileEntry->md5 && fileEntry->md5[0] != '\0'  && fileEntry->md5[1] == ':') {
+		switch (fileEntry->md5[0]) {
+		case 'r':
+			return kMD5MacResFork;
+		case 'x':
+			return (MD5Properties) (kMD5MacResFork | kMD5Tail);
```

Comment:
What about using capital letters for the tail? e.g. 'r'/'R', 'd'/'D'? Or a combination of letters like 'rt:', because there is no easy-to-remember way what 'x:' or 'z:' stands for. Developers are filling these tables manually, so it must be intuitive.

Reply:
Changed to rt: